crates/pm_auth/Cargo.toml

```diff
@@ -21,6 +21,7 @@ bytes = "1.4.0"
 error-stack = "0.4.1"
 http = "0.2.9"
 mime = "0.3.17"
+reqwest = "0.12.2"
```

Comment:
Can we avoid pulling one more dependency here, maybe bump `http` to `1.x` instead?